Source/WebCore/platform/graphics/cg/GraphicsContextCG.h

```diff
@@ -152,12 +152,12 @@ class WEBCORE_EXPORT GraphicsContextCG : public GraphicsContext {
     CGContextRef contextForState() const;
 
     const RetainPtr<CGContextRef> m_cgContext;
-    const RenderingMode m_renderingMode;
-    const bool m_isLayerCGContext;
-    mutable bool m_userToDeviceTransformKnownToBeIdentity { false };
-    // Flag for pending draws. Start with true because we do not know what commands have been scheduled to the context.
-    bool m_hasDrawn { true };
     mutable std::optional<DestinationColorSpace> m_colorSpace;
+    const RenderingMode m_renderingMode : 1; // NOLINT
+    const bool m_isLayerCGContext : 1;
+    mutable bool m_userToDeviceTransformKnownToBeIdentity : 1 { false };
+    // Flag for pending draws. Start with true because we do not know what commands have been scheduled to the context.
+    bool m_hasDrawn : 1 { true };
```

Comment:
Bit of an overkill perhaps for a type that won't have many instances alive? Isn't this a stack allocated type anyway?

Reply:
No, heap allocated.
If we have 20000 layers or 2d contexts, we use 160 mb of GPUP's 300mb limit..